components/intake.py

```diff
@@ -1,22 +1,40 @@
-from magicbot import tunable
+from magicbot import feedback, tunable
 from phoenix5 import ControlMode, TalonSRX
 
 from ids import TalonId
 
 
 class IntakeComponent:
     intake_output = tunable(0.9)
+    deploy_output = tunable(0.1)  # Test values
+    retract_output = tunable(-0.1)
 
     def __init__(self) -> None:
         self.motor = TalonSRX(TalonId.INTAKE)
         self.motor.setInverted(False)
 
+        self.deploy_motor = TalonSRX(TalonId.DEPLOY_INTAKE)
+
         self.desired_output = 0.0
+        self.desired_deploy_output = 0.0
 
     def intake(self):
         self.desired_output = self.intake_output
 
+    def deploy(self) -> None:
+        self.desired_deploy_output = self.deploy_output
+
+    def retract(self) -> None:
+        self.desired_deploy_output = self.retract_output
+
+    @feedback
+    def deploy_motor_current(self) -> float:
+        return TalonSRX.getSupplyCurrent(self.deploy_motor)
```

Comment:
This will work, but is there a reason you're calling an instance method via the class?
```suggestion
        return self.deploy_motor.getSupplyCurrent()
```

Reply:
It's because my brain is made of cardboard and sticky tape